pkg/kube/client.go

```diff
@@ -369,15 +407,186 @@ func (c *client) NewPortForwarder(podName, ns, localAddress string, localPort in
 	return newPortForwarder(c.config, podName, ns, localAddress, localPort, podPort)
 }
 
-func (c *client) PodsForSelector(ctx context.Context, namespace, labelSelector string) (*v1.PodList, error) {
-	podGet := c.Get().Resource("pods").Namespace(namespace).Param("labelSelector", labelSelector)
-	obj, err := podGet.Do(ctx).Get()
+func (c *client) PodsForSelector(ctx context.Context, namespace string, labelSelectors ...string) (*kubeApiCore.PodList, error) {
+	return c.CoreV1().Pods(namespace).List(ctx, kubeApiMeta.ListOptions{
+		LabelSelector: strings.Join(labelSelectors, ","),
+	})
+}
+
+func (c *client) ApplyYAMLFiles(namespace string, dryRun bool, yamlFiles ...string) error {
```

Comment:
Nit: I think I would prefer dry run to be its own function so we don't need to pollute the call with `false` everywhere? Dry run is pretty rare anyways

Reply:
done.